log_transfer/models.py

```diff
@@ -1,7 +1,9 @@
 from django.conf import settings
+from django.contrib.auth.models import User
```

Comment:
To me this looks like unused import, is this actually needed or added accidentally?

Reply:
It is, I used it to test at some point and forgot it. I'll remove it.

Reply:
Done.